### DISCUSSION TITLE
Terraform Script to Azure

### DISCUSSION THREAD

**Discussion by akpotluri23** [emqx/emqx#11578] [General]
[](url)Good morning all,
 
I was working on deploying the terraform to azure and was following up on the steps as listed on the GitHub when I ran into an error. Here is the command I’ve been running:
terraform apply -auto-approve -var="is_emqx5=true" -var="emqx5_core_count=2" -var="emqx_vm_count=3" -var="emqx5_package=https://www.emqx.com/en/downloads/broker/5.1.6/emqx-5.1.6-ubuntu22.04-amd64.deb"
The error log is also attached below as an image and was hoping to see if you guys had any insight on that?
 
[Thanks,](![emqxSupport](https://github.com/emqx/emqx/assets/137217450/b2bc5d1f-6350-49d9-8ece-708d29decb38))

**Comment by akpotluri23** [comment]
![emqxSupport](https://github.com/emqx/emqx/assets/137217450/50ccac50-358b-4574-956f-dd08c0eea831)

Sorry here is the error log.

**Reply by jacky-xbb** [reply]
Using  the following command:
 
`terraform apply -auto-approve -var="is_emqx5=true" -var="emqx5_core_count=2" -var="emqx_vm_count=3" -var="emqx5_package=https://www.emqx.com/en/downloads/broker/5.1.6/emqx-5.1.6-ubuntu20.04-amd64.tar.gz"`
 
 
 
Since currently we only support .tar.gz and .zip file extensions, only Ubuntu 20.04 has been verified.

**Reply by akpotluri23** [reply]
Hello, @jacky-xbb, I tried running the above and and have run into the exact same error. I have attached the error log below again. 

![image](https://github.com/emqx/emqx/assets/137217450/e382e5be-1b18-48bd-9981-9702f984f9d4)
